src/redux/__test__/reducer.test.ts

```diff
@@ -0,0 +1,309 @@
+import reducer, { initialState } from '../reducer';
+import {
+  CREATE_MOVIE,
+  CREATE_MOVIE_FAIL,
+  CREATE_MOVIE_SUCCESS,
+  GET_MOVIE_DETAILS,
+  GET_MOVIE_DETAILS_FAIL,
+  GET_MOVIE_DETAILS_SUCCESS,
+  EDIT_MOVIE,
+  EDIT_MOVIE_FAIL,
+  EDIT_MOVIE_SUCCESS,
+  GET_MOVIES,
+  GET_MOVIES_FAIL,
+  GET_MOVIES_SUCCESS,
+  REMOVE_MOVIE,
+  REMOVE_MOVIE_FAIL,
+  REMOVE_MOVIE_SUCCESS,
+} from '../types';
+
+describe('Reducer unit testing', () => {
+  describe('get movies types', () => {
+    test('GET_MOVIES ', () => {
+      expect(reducer(initialState, { type: GET_MOVIES, payload: { params: '' } })).toEqual({
+        ...initialState,
+        isMoviesLoading: true,
+        error: null,
+      });
+    });
+
+    test('GET_MOVIES_SUCCESS', () => {
+      expect(
+        reducer(initialState, {
+          type: GET_MOVIES_SUCCESS,
+          payload: [
+            {
```

Comment:
Can we create a separate variable for this? It looks the same as on the lines 55-67?

Reply:
yep, it makes sense,
on my current project dev led said about this strategy to avoid  modification mocks in the future and for better-understanding guys who review your code